docs/packages.md

```diff
@@ -24,6 +24,22 @@ The Linux Software Repository can be configured to automatically install the Lin
 > Optionally, if you prefer manual configuration, the Linux Software Repository configuration files are available at [https://packages.microsoft.com/config](https://packages.microsoft.com/config/). The name and location of these files can be located using the following URI naming convention:
 `https://packages.microsoft.com/config/<Distribution>/<Version>/prod.(repo|list)`
 
+### Directory structure
+
+At the root of packages.microsoft.com, you’ll see folders like:
+
+- `/config/<distro>/<version>/prod.list`: Contains repo configuration files for each supported distribution.
```

Comment:
- `/config/<distro>/<version>/packages-microsoft-prod.[rpm|deb]`: Provides the `.list` or `.repo` file to enable the distribution-specific repository for each system, as well as the appropriate signing key. If more a manual or non-default (for example, enabling the early-access not-yet-production "insiders" repos) approach is desired then individual `.list` or `.repo` files are available in this directory.